journal/src/main/java/io/zeebe/journal/file/SegmentedJournal.java

```diff
@@ -552,15 +552,12 @@ void resetHead(final long index) {
   /**
    * Resets journal readers to the given tail.
    *
-   * @param index The index at which to reset readers.
+   * @param deletedIndex The index at which to reset readers.
    */
-  void resetTail(final long index) {
+  void resetTail(final long deletedIndex) {
```

Comment:
Two small nits, feel free to not apply them if you disagree: `resetAdvancedReaders` or something would be more descriptive, `resetTail` doesn't really clarify the method. `deletedIndex` isn't meaningful in the context of this method, it's just an index